### ISSUE TITLE
Extensions on body and form parameters are not retained

### ISSUE THREAD

**Issue by padamstx** [Mermade/oas-kit#112]
When I use swagger2openapi (command line), extensions defined on body and form parameters within a swagger 2.0 apispec are not copied to the resulting openapi 3 apispec.
Note: I am currently using swagger2openapi v3.2.13

### Checklist

- [X] Conversion: I have checked my source definition is valid OpenAPI 2.0
- [X] Conversion: On valid OpenAPI 2.0 input, the conversion looks wrong

### Detailed Description
I have a swagger 2.0 apispec which contains extensions in various places.   The extensions set on body and form parameters seem to be lost in the conversion process.   I would have expected that extensions set on a body param would be carried forward and set on the resulting requestBody object.   For extensions set on form params, I would expect those to be carried forward and set on the schema property which corresponds to the form param.

Here is a small swagger 2.0 apispec which can serve as a testcase:

```
{
    "swagger":"2.0",
    "info":{
        "title":"Testcase for swagger2openapi issue",
        "description":"This is a testcase which demonstrates a problem in swagger2openapi.  Specifically, extension properties defined on body and form parameters are not carried forward to the requestBody object within the converted apispec.",
        "version":"1.0.0",
        "x-info-extension":"Example of an extension defined for the 'info' object."
    },
    "x-root-extension": "Example of an extension set at the root level.",
    "paths":{
        "/testcase/v1/person":{
            "post":{
                "summary":"Body param test #1",
                "description":"Testcase that uses a body param which has extensions set on it.",
                "operationId":"createPerson",
                "consumes":[
                    "application/json"
                ],
                "parameters":[
                    {
                        "name":"person",
                        "in":"body",
                        "required":true,
                        "description":"The person to be created.",
                        "schema":{
                            "$ref":"#/definitions/Person"
                        },
                        "x-body-extension":"This is an example of an extension defined on a body parameter."
                    }
                ],
                "responses":{
                    "201":{
                        "description":"Created"
                    },
                    "default":{
                        "description":"An error occurred."
                    }
                },
		"x-operation-extension": "Example of an extension set on the 'createPerson' operation."
            }
        },
        "/testcase/v1/forms":{
            "post":{
                "summary":"Form param test #1",
                "description":"Testcase that uses form params which have extensions set on them.",
                "operationId":"createForms",
                "consumes":[
                    "multipart/form-data"
                ],
                "parameters":[
                    {
                        "name":"stringPart",
                        "in":"formData",
                        "required":true,
                        "description":"A form part which is a string.",
                        "type":"string",
                        "x-alternate-name":"alternateStringPart"
                    },
                    {
                        "name":"filePart",
                        "in":"formData",
                        "required":true,
                        "description":"A form part which is a file.",
                        "type":"file",
                        "x-file-content-types":[
                            "application/octet-stream",
                            "application/zip"
                        ],
                        "x-include-filename":true,
                        "x-alternate-name":"alternateFilePart"
                    }
                ],
                "responses":{
                    "201":{
                        "description":"Created"
                    },
                    "default":{
                        "description":"An error occurred."
                    }
                },
		"x-operation-extension": "Example of an extension set on the 'createForms' operation."
            }
        }
    },
    "definitions":{
        "Person":{
            "type":"object",
            "description":"Models a person.",
            "properties":{
                "name":{
                    "type":"string",
                    "description":"Person's name."
                },
                "age":{
                    "type":"integer",
                    "description":"Person's age."
                }
            },
	    "required": ["name", "age"]
        }
    }
}
```

**Comment by MikeRalphson** [comment]
Thanks for the detailed issue. Will take a look.

**Comment by padamstx** [comment]
cc: @mkistler

**Comment by padamstx** [comment]
@MikeRalphson, perhaps you could assign this issue to me as I'll include the fix for it in my PR that also implements #113

**Comment by MikeRalphson** [comment]
Thanks!
